static/frontary/pumpkin/theme.css

```diff
@@ -1886,9 +1897,11 @@ div.input-select-vector {
 }
 
 div.input-select-vector-vec {
-    display: flex;
-    align-items: center;
-    justify-content: center;
+    display: block;
+}
+
+div.searchable-select-vector>div {
+    width: 100% !important;
```

Comment:
Is the use of `!important` unavoidable in this case? My understanding is that, for Frontary to provide reusable and customizable components, I am afraid that using `!important` can limit flexibility by making it harder for library users to override styles for specific use cases. I’d appreciate learning the rationale behind using `!important` here to better understand and align with this choice.

Reply:
I'm also aware that `!important` should be used to the minimum required. If I don't use `!important`, I think it will be inevitable to change the DOM structure, so I'll think about it a little more and open PR again. I won't think too long and make it reviewable again within today.

Reply:
I got it! I found a way to resolve this issue without making significant changes to the Frontary. As a result, the only change in this PR is the removal of `!important`. I removed all '!important' used in CSS and modified them to control width width with props in the UI repository using them.